### DISCUSSION TITLE
Commenting feature request

### DISCUSSION THREAD

**Discussion by mdow-tx** [outline/outline#3492] [General] [answered]
It'd be great if Outline had a commenting thread optionally below documents. For example, if you're submitting an [RFC](https://en.wikipedia.org/wiki/Request_for_Comments) for your company's new policy or procedure, having the ability to comment below the document would be great. Confluence has the feature and it is very helpful to gain feedback. 

We do use Slack and can have channels or threads about documents there as well. If someone has a good idea to implement that let me know.

**Comment by strawberryfly** [comment, marked as answer]
Outline will add a comment function in version 0.65 in mid-June (as of today). You can have a look at it at [Comment#457](https://github.com/outline/outline/issues/457)

**Reply by Limezy** [reply]
However, as far as I understood the plan is to have inline comments, anchored to a paragraph or to a word, not below the documents

**Reply by mdow-tx** [reply]
> However, as far as I understood the plan is to have inline comments, anchored to a paragraph or to a word, not below the documents

That's fine and good, just seems like more people would benefit from a comment thread below the doc than inline comments. Not sure. Maybe my use case is different.

**Reply by Limezy** [reply]
For me both below the doc and inline will be very useful. Actually it's the only missing feature I need before my project can go live.
I also hope there will be in the near future a feature to track changes and comment these changes !
